### ISSUE TITLE
Text Links Contrast Ratio

### ISSUE THREAD

**Issue by akamkar** [contiv/contiv.github.io#71]
The blue color of text links is too light and doesn't have enough contrast. It failed the accessibility test. Please replace it with #036e97. 
Also, if could we use dark gray (#5c5d61 ) for text to differentiate the text from links?
![screen shot 2017-02-15 at 1 47 19 pm](https://cloud.githubusercontent.com/assets/18014543/22996964/7a4f139e-f385-11e6-9f10-61735da11da4.png)

**Comment by stacykor** [comment]
Yikes. This has been updated to meet accessibility standards.
